### PR TITLE
Added single-actor and actors-single.scss

### PR DESCRIPTION
**Pull request by PJWSTKs16479**

### CODE REVIEW

File: custom-fields.php [hunk 1]
```diff
@@ -31,4 +31,50 @@ function filmy_field_groups() {
...
+                'name' => 'gender',
+                'type' => 'text',
+            ),
+            array (
+                'key' => 'field_birthday_date',
+                'label' => 'Data urodzenia',
+                'name' => 'birthday_date',
+                'type' => 'date_picker',
+            ),
+
```
Comment (paskowski) [inline custom-fields.php:60]: That space empty line could be deleted

File: package.json [hunk 2]
```diff
@@ -18,5 +18,8 @@
   "homepage": "https://github.com/paskowski/movies-cms#readme",
   "devDependencies": {
     "node-sass": "^5.0.0"
+  },
```
Comment (paskowski) [inline package.json:21]: I'd revert this change. We have bootstrap already added manually, I see no point in adding it this way.

File: single-actor.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            </div>
+
+            <div class="white_column">
+              
+                <div class="title" dir="auto">
+                  <h1><?php the_title(); ?></h1>
+                </div>
+              
+              <section>
+                <h3 dir="auto">Biografia</h3>
```
Comment (paskowski) [inline single-actor.php:31]: I don't think we need dir, we're not going to support any other languages than polish

File: single-actor.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            <div class="grey_column">
+              
+                <div>
+                <?php $image = get_field('profile_image'); ?>
+                    <img src="<?php echo esc_url($image['url']); ?>" alt='<?php echo esc_attr($image['alt']); ?>'/>
+                </div>
+              
+              <div class="column">
+                
+                  <h3><bdi>Informacje osobiste</bdi></h3>
```
Comment (paskowski) [inline single-actor.php:13]: I'd say we should use css for styling instead of strong. For bdi, as with dir, we're not going to use any non LTR languages so I would remove it for brevity.

**Review by paskowski — COMMENTED** [review]
Looks good 🎉! Left few comments on what I think we can improve.

**Review by MaciejSikorski — APPROVED** [review]
